forms/constants.go

```diff
@@ -1,7 +1,15 @@
 package forms
 
+const ContentTypeCSV = "text/csv"
+const ContentMinimumSize = 1
+const ContentMaximumSize = 1000
```

Comment:
How about being more explicit as I was not sure where `ContentTypeCSV` was coming from the code below 😓 

```
const KeywordUploadContentTypeCSV = "text/csv"
const KeywordUploadMinimumSize = 1
const KeywordUploadMaximumSize = 1000
```